app/assets/stylesheets/styles.css

```diff
@@ -26,6 +26,11 @@
   line-height: 2.2;
 }
 
+#new-bath-form input{
```

Comment:
Nice design choice! I actually also just learned something from this. Class vs. Id selection. Thanks!